### PR TITLE
Add Connectors Monitoring

### PR DESCRIPTION
**Pull request by tomasz-sadura**
Add section on how to monitor Managed Connectors using Prometheus and Grafana.

### CODE REVIEW

**Review by bochenekmartin — APPROVED** [review]

**Comment by vladoschreiner** [comment]
Review, please address before merging:

1. The preview site doesn't render well: https://deploy-preview-1749--redpanda-documentation.netlify.app/docs/deploy/deployment-option/cloud/managed-connectors/monitor-connectors/
2. The step-by-step experience in the docs doesn't flow well when following the _Running the Prometheus and Grafana locally_ guideline. Explain when to follow the readme and when to stay in the docs.
3. In my test case, the dashboard didn't show a connector with failed tasks only (MM2 pointed to non-existing source). Is it a bug or a known limitation? If so, please elaborate on the known limitations.

**Review by bochenekmartin — APPROVED** [review]

**Comment by tomasz-sadura** [comment]
> The preview site doesn't render well: https://deploy-preview-1749--redpanda-documentation.netlify.app/docs/deploy/deployment-option/cloud/managed-connectors/monitor-connectors/

Fixed.

> The step-by-step experience in the docs doesn't flow well when following the Running the Prometheus and Grafana locally guideline. Explain when to follow the readme and when to stay in the docs.

This documentation tells how to configure (any) Prometheus and import Grafana dashboard.
The guide in the observability repo on the other hand treats how to run Prometheus and Grafana locally.
The guide is linked here as a quick-start. User does not need to follow it, can self-host services instead.
Do you prefer this documentation to be opinionated for the local run use-case?

> In my test case, the dashboard didn't show a connector with failed tasks only (MM2 pointed to non-existing source). Is it a bug or a known limitation? If so, please elaborate on the known limitations.

Could you share the exact config? If dashboard failed to show failed tasks, then it is a bug.
In case when connector did not create any tasks, no task metrics are reported, so that dashboard show no tasks.
Added this info to `Limitations` section.

File: docs/deploy/deployment-option/cloud/managed-connectors/monitor-connectors.mdx [hunk 1]
```diff
@@ -0,0 +1,128 @@
+---
+title: Connectors monitoring
```
Comment (Feediver1) [inline docs/deploy/deployment-option/cloud/managed-connectors/monitor-connectors.mdx:2]:
```suggestion
title: Monitor Connectors
```

File: docs/deploy/deployment-option/cloud/managed-connectors/monitor-connectors.mdx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+<head>
+    <meta name="title" content="Connectors monitoring | Redpanda Docs"/>
+    <meta name="description" content="Monitor Managed Connectors with metrics."/>
+    <link rel="canonical" href="https://docs.redpanda.com/docs/deploy/deployment-option/cloud/managed-connectors/monitor-connectors/" />
+</head>
+
+
+Redpanda exports metrics for Managed Connectors through a Prometheus HTTPS endpoint for you to monitor connectors health.
+You can use Prometheus and Grafana to visualize the metrics and set up alerting.
```
Comment (Feediver1) [inline docs/deploy/deployment-option/cloud/managed-connectors/monitor-connectors.mdx:13]:
```suggestion
You can monitor connector health using the managed connector metrics, which Redpanda exports through a Prometheus HTTPS endpoint. You can use Prometheus and Grafana to visualize the metrics and set up alerts.
```

File: docs/deploy/deployment-option/cloud/managed-connectors/monitor-connectors.mdx [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+## Configure Prometheus
+
+[Prometheus](https://prometheus.io/) is a system monitoring and alerting tool. It collects and stores metrics as time-series data identified by a metric name and key/value pairs.
+
+:::note
+You can quickly start by [running the Prometheus and Grafana locally](https://github.com/redpanda-data/observability/tree/main/cloud).
```
Comment (Feediver1) [inline docs/deploy/deployment-option/cloud/managed-connectors/monitor-connectors.mdx:20]:
```suggestion
You can quickly start by [running Prometheus and Grafana locally](https://github.com/redpanda-data/observability/tree/main/cloud).
```

**Comment by vladoschreiner** [comment]
Minor comment: In the preview, I can't locate the page using the navigation (left-hand menu). E.g. it's orphaned. That should be fixed. The page should be co-located either with Managed connectors or in the Monitoring section of the docs.

**Comment by Feediver1** [comment]
Many edits reqd--rather than forking, etc. I am opening a new PR.

**Comment by vladoschreiner** [comment]
@Feediver1 Where is the PR, please?

**Comment by tcoupland** [comment]
replaced by https://github.com/redpanda-data/documentation/pull/1862

**Pull request closed without merging**